### PR TITLE
[stable/prometheus-operator] Add docker checksum option

### PR DESCRIPTION
**Pull request by GMartinez-Sisti**
#### What this PR does / why we need it:
- Adds optional docker checksum option to improve security for the deployed containers

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by mrueg** [comment]
Isn't it easier to include the checksum in the tag value of each in image in your values.yaml? 

The tag in values.yaml of the chart `v1.0.1` simply becomes `v1.0.1@sha256:checksum`

**Comment by GMartinez-Sisti** [comment]
@mrueg That was my first approach too, but didn't work:
- The chart is validating the semver of prometheus-operator, so it won't accept that
- Prometheus-operator supports natively specifying the sha in the [AlertmanagerSpec](git@github.com:helm/charts.git) and [PrometheusSpec](git@github.com:helm/charts.git) configs

I think this way it makes it wasier to understand how to use the chart values.

File: stable/prometheus-operator/README.md [hunk 1]
```diff
@@ -193,6 +193,7 @@ The following tables list the configurable parameters of the prometheus-operator
 | `prometheusOperator.admissionWebhooks.patch.image.pullPolicy` | Image pull policy for the webhook integration jobs | `IfNotPresent` |
 | `prometheusOperator.admissionWebhooks.patch.image.repository` | Repository to use for the webhook integration jobs | `jettech/kube-webhook-certgen` |
 | `prometheusOperator.admissionWebhooks.patch.image.tag` | Tag to use for the webhook integration jobs | `v1.2.1` |
+| `prometheusOperator.admissionWebhooks.patch.image.sha` | Sha to use for the webhook integration jobs (optional) | `3e6adf0140f5749427a83a750cc436c532872fa7bf01d01b25bb163737dab245` |
```
Comment (vsliouniaev) [inline stable/prometheus-operator/README.md]: Could we drop the default SHA configurations? Makes updates less of a headache and I'm not sure this is something that I would want the chart to provide for me if I wanted to set this configuration.

**Review by vsliouniaev — CHANGES_REQUESTED** [review]
/ok-to-test

File: stable/prometheus-operator/README.md (hunk 1 above)
Comment (GMartinez-Sisti) [reply stable/prometheus-operator/README.md]: Sure, change pushed.

**Pull request merged**